src/main/java/frc/robot/Constants.java

```diff
@@ -1,10 +1,50 @@
 package frc.robot;
 
 
+import edu.wpi.first.math.MatBuilder;
+import edu.wpi.first.math.Matrix;
+import edu.wpi.first.math.Nat;
+import edu.wpi.first.math.numbers.N1;
+
 public final class Constants {
+    public static final double LOOP_PERIOD = 0.02; // loop period. [s]
+    public static final double g = 9.80665; // Gravity acceleration constant. [m/s^2]
+    public static final double UPPER_TARGET_HEIGHT = 2.64; // Height of upper target. [m]
+    public static final double PEBZNER_HEIGHT = 4.8; // Height of pebzner auditorium. [m]
+    public static final int TALON_TIMEOUT = 10; // Waiting period for configurations [ms].
```

Comment:
See comment for `LOOP_PERIOD`. Also, I do recommend using the timeout when configuring motor controllers, but @adamv17 needs to decide.